handlers/lookup_test.go

```diff
@@ -92,23 +92,26 @@ var _ = Describe("Lookup", func() {
 			reg.LookupReturns(pool)
 		})
 
-		It("sends a bad request metric", func() {
-			Expect(rep.CaptureBadRequestCallCount()).To(Equal(1))
+		It("does not send a bad request metric", func() {
+			Expect(rep.CaptureBadRequestCallCount()).To(Equal(0))
 		})
 
-		It("Sets X-Cf-RouterError to unknown_route", func() {
-			Expect(resp.Header().Get("X-Cf-RouterError")).To(Equal("unknown_route"))
+		It("Sets X-Cf-RouterError to no_endpoints", func() {
```

Comment:
I know that this was in the original code, but can you please change `Sets` to `sets`?

Reply:
Done in https://github.com/cloudfoundry/gorouter/pull/251/commits/8c0eb34791893c96976965c37237241d639c4c49